src/Dofs/MixedDofHandler.jl

```diff
@@ -0,0 +1,254 @@
+
+mutable struct FieldHandler
+    field_names::Vector{Symbol}
+    field_dims::Vector{Int}
+    field_interpolations::Vector{JuAFEM.Interpolation}
+    quad_rules::Vector{JuAFEM.QuadratureRule}
```

Comment:
I dont think QuadratureRule should be stored here. User should handle this on their own?

Reply:
Yes, I agree. I don't know why I added it there.